parser/sql/dialect/doris/src/main/antlr4/imports/doris/DALStatement.g4

```diff
@@ -411,6 +411,18 @@ restart
     : RESTART
     ;
 
+recoverDatabase
+    : RECOVER DATABASE databaseName (databaseId | AS newDatabaseName)?
```

Comment:
You can use [rule-element-labels](https://github.com/antlr/antlr4/blob/master/doc/parser-rules.md#rule-element-labels) instead of the newDatabaseName rule.

Reply:
Done